### ISSUE TITLE
Sometimes setup transports are suggested in route finder.

### ISSUE THREAD

**Issue by evanlinjin** [skycoin/skywire-testnet#363]
Reproduction:
1. Run [`run-generic-env.sh`](https://github.com/watercompany/skywire-services/blob/master/integration/run-generic-env.sh).
2. Run the following in the `shell` tab:
    ```
    $ CLI_A node add-tp $SN_PK && CLI_B node add-tp $SN_PK && CLI_C node add-tp $SN_PK && CLI_A node add-tp $PK_B && CLI_C node add-tp $PK_B
    # Okay here.

    $ ./integration/test-messaging.sh
    # route setup: RespFailure, packet type: Unknown(254)
    # route setup: RespFailure, packet type: Unknown(254)
    ```

This is probably caused because:
- `setup-node` allows others to register transports with itself as `public`.
- `skywire-node` allows registering transports with setup nodes as `public`.

**Comment by ayuryshev** [comment]
IIRC: after restart of NodeA, NodeB, NodeC this problem disappears.
And the real problem is that transports are not working right after `node add-tp` command

**Comment by ayuryshev** [comment]
And to reproduce this problem in 100% one must run `./integration/tear-down.sh` before starting `./integration/run-generic-env.sh`.

**Comment by evanlinjin** [comment]
> IIRC: after restart of NodeA, NodeB, NodeC this problem disappears.
> And the real problem is that transports are not working right after `node add-tp` command

@ayuryshev  From my understanding, and also what I wrote in this issue; the reason why transports are not working after `node add-tp` is because:
- `setup-node` allows others to register transports with itself as public.
- `skywire-node` allows registering transports with setup nodes as public.

And hence:
- Setup transports are suggested in route finder.

If I am wrong about this, please advise otherwise.

**Comment by evanlinjin** [comment]
@ayuryshev I also tried restarting all three nodes, and the problem still persists. I believe this is because setup transports are used for routes.

<img width="683" alt="image" src="https://user-images.githubusercontent.com/14065705/58178905-a8029600-7cd9-11e9-9a22-dc70e739b511.png">

**Comment by jdknives** [comment]
This will be closed by #443 if I understand correctly, right? @evanlinjin @nkryuchkov

**Comment by jdknives** [comment]
As far as I understand, this has bee fixed by the recent PR that removes the usage of transport.manager from the setup node. Is that correct? If so, please close this issue. @evanlinjin
